### DISCUSSION TITLE
Number of tabs in a tab group displayed

### DISCUSSION THREAD

**Discussion by FreePDF64** [okaryo/TabTabTab#89] [Ideas]
My next wish for TabTabTab is:
I would like to see the number of tabs in a tab group displayed. Because I use AddOns for the automatically creation of tab groups - and it would be helpful if I can see into TabTabTab how many tabs inside a tab group.

**Comment by okaryo** [comment]
Thanks for your suggestion. I already have plans for that feature and it's been issued in https://github.com/okaryo/TabTabTab/issues/40 . I'm currently working on syncing the open/close state of groups in https://github.com/okaryo/TabTabTab/issues/87 , and I plan to address this feature while I'm at it.

**Comment by okaryo** [comment]
The feature has been implemented with the release of version [2.15.0](https://github.com/okaryo/TabTabTab/releases/tag/2.15.0) 🚀!
Please wait a little longer for it to become publicly available as it is currently under store review.

Also, if anything feels off when you use it, please give me feedback!

**Comment by FreePDF64** [comment]
I am totally thrilled. The implementation is perfectly done!

Very very good job!

**Comment by FreePDF64** [comment]
Hi,
I discovered a problem: see attached picture.

The counter from "currrent window" shows a wrong number (here 3). I have open a group with 2 Tabs - and two other Tabs not inside a group: Total 4 Tabs.

![TabTabTab](https://github.com/okaryo/TabTabTab/assets/38452654/b5e20023-3ad0-4956-948d-929c99b59bfd)

**Reply by okaryo** [reply]
Thanks for sharing! I've identified the issue, and I'll fix it right away!

**Comment by okaryo** [comment]
Fixed in [v2.15.3](https://github.com/okaryo/TabTabTab/releases/tag/2.15.3)! Please wait for the store review to be completed!
